allensdk/brain_observatory/behavior/swdb/behavior_project_cache.py

```diff
@@ -187,12 +294,49 @@ def get_stimulus_presentations(self):
         # Rename some columns to make more sense to students
         stimulus_presentations = stimulus_presentations.rename(
             columns={'index':'absolute_flash_number'})
+        # Replace image set with A/B
+        stimulus_presentations['image_set'] = self.get_task_parameters()['stage'][15]
+        # Change index name for easier merge with flash_response_df
+        stimulus_presentations.index.rename('flash_id', inplace=True)
         return stimulus_presentations
 
     def get_stimulus_templates(self):
+        # super stim templates is a dict with one annoyingly-long key, so pop the val out
         stimulus_templates = super(ExtendedNwbApi, self).get_stimulus_templates()
         return stimulus_templates[list(stimulus_templates.keys())[0]]
 
+    def get_segmentation_mask_image(self):
+        # We need to binarize the segmentation mask image. Currently ROIs have values
+        # between 0 and 1, but it is unclear what the values are and this will be 
+        # confusing to students.
+        segmentation_mask_itk = super(ExtendedNwbApi, self).get_segmentation_mask_image()
+        segmentation_mask_image = ImageApi.deserialize(segmentation_mask_itk)
+        segmentation_mask_image.data[segmentation_mask_image.data > 0] = 1
+        segmentation_mask_itk = ImageApi.serialize(data=segmentation_mask_image.data,
+                                                   spacing=segmentation_mask_image.spacing,
+                                                   unit=segmentation_mask_image.unit)
+        return segmentation_mask_itk
+
+    def get_licks(self):
```

Comment:
went over htis in person, current state is that licks is a dataframe with a single column: time & rewards is a dataframe with time column + other data columns. Maybe would be better to have licks as ndarray and rewards back to time-indexed? Otherwise we have some meaningless indices floating around,